app/models/postit.rb

```diff
@@ -20,13 +20,17 @@ def initialize(new_postit)
     attr_reader :id, :longitude, :latitude, :message, :created_at
 
     def to_json(_options = {})
+      to_map.to_json
+    end
+
+    def to_map
```

Comment:
map are called hash in ruby, maybe rename it to to_h ?

reference
https://blog.appsignal.com/2018/09/25/explicitly-casting-vs-implicitly-coercing-types-in-ruby.html

Reply:
You're right, I'll fix it later.